### ISSUE TITLE
Popup element of the selects should have aria-hidden attribute

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-ui-core#3380]
### Bug report

[http://demos.telerik.com/kendo-ui/dropdownlist/index](http://demos.telerik.com/kendo-ui/dropdownlist/index)

### Reproduction of the problem
1. Find the k-list-container element
2. Notice that it's announced by the screen readers even though it has display: none

### Environment

* **Kendo UI version:** 2017.2.504
* **Browser:**all
